.github/workflows/lint-test.yml

```diff
@@ -23,13 +23,19 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Setup Poetry 🔧
+        uses: ./.github/actions/poetry
```

Comment:
Instead of copying this file into this repo, you should just be able to reference the file in the other repo. 

Something like this: 
```
      - name: Setup Poetry 🔧
        uses: ThePalaceProject/circulation/.github/actions/poetry@main
```

Reply:
This allows a different version of poetry if required for this repo

Reply:
Also keeps it self sustained, in case the name/branch/version/host of the circ repo changes